src/autolaser.py

```diff
@@ -73,12 +107,12 @@ def find_center_point(max_contour):
 
     write_read_arduino(f"{180-x_angle}x") #Send the x-angle
     write_read_arduino(f"{y_angle}y") #Send the y-angle
-
     return center
 
 
 # Adds an outline around the balloon and adds a center dot on the frame displayed to users
 def draw_outline(max_contour, frame):
+    global radius
     center = None
     ((x, y), radius) = cv2.minEnclosingCircle(max_contour)
```

Comment:
correct, the reason for making the radius global is so that we can call the distance function from anywhere else in the code (since I wasn't sure where we should put it)

Reply:
I removed the global radius and put the distance function call in the draw_outline function